tests/backend/test_backend_reference.py

```diff
@@ -18,6 +18,9 @@
 from onnx import ModelProto
 from onnx.backend.base import Device, DeviceType
 from onnx.reference import ReferenceEvaluator
+from onnx.test import version_utils
```

Comment:
Is onnx.test part of the package? Should we create a copy of  version_utils?

Reply:
It was literally a one-liner so I just copied it and removed this import.